10up/classifai
# Add notice for deprecated IBM Watson endpoint.

### Description of the Change


Add a notice in readme and to Language Processing settings for deprecated IBM Watson endpoint.

### Alternate Designs



### Benefits


When user set deprecated endpoint for IBM Watson NLU API URL with `watsonplatform.net` in url, they'll see a notice about endpoint is deprecated and a link to guide on how to use/get new url.

### Possible Drawbacks



### Verification Process



1. Go to Language Processing Settings.
2. Add API url with `watsonplatform.net` endpoint.
3. You'll see an error/notice deprecated endpoint message as shown in below screenshot:

     <img width="1258" alt="image" src="https://user-images.githubusercontent.com/10358350/145406210-3f53bffc-e768-4647-aeea-9112c3f529eb.png">

4. Update this URL with valid pattern i.e `api.us-south.natural-language-understanding.watson.cloud.ibm.com`
5. You won't see this deprecated endpoint message anymore.

     <img width="1261" alt="image" src="https://user-images.githubusercontent.com/10358350/145406573-23fe9ff9-290d-477d-b2c0-d0c618bfd08f.png">

### Checklist:






### Applicable Issues



Closes: https://github.com/10up/classifai/issues/316

### Changelog Entry



- Add notice for deprecated IBM Watson endpoint.
